wire-ios-sync-engine/Tests/Source/UserSession/ZMUserSessionTests+AccessToken.swift

```diff
@@ -16,11 +16,14 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-import Foundation
-@testable import WireSyncEngine
 import WireTransport
 import XCTest
```

Comment:
```suggestion
import XCTest
```
see the SwiftLint error below